crates/spicepod/src/component/dataset.rs

```diff
@@ -131,6 +131,28 @@ impl Dataset {
         }
     }
 
+    #[must_use]
+    pub fn acceleration_params(&self) -> Option<HashMap<String, String>> {
+        if let Some(acceleration) = &self.acceleration {
+            if let Some(refresh_interval) = &acceleration.params {
```

Comment:
You can just return `acceleration.params.clone()` instead of unwrapping it and then rewrapping it in an optional.

Reply:
Thanks, I updated this part and will push the changes in future commit